### PR TITLE
Change in UUID fetching for first launches

### PR DESCRIPTION
**Pull request by nethip**
With this PR, UUID is now mapped to a system hash instead of relying on unique identifiers. This will make sure the user id is captured uniquely and not logged as two users in situations like launching Brackets after clearing preferences.

Also fixed linter errors in this file.

[This](https://github.com/adobe/brackets-shell/pull/603) relevant PR on Brackets-shell side needs to be merged too. The PR on the shell side introduces new shell API by name `getMachineHash`

Pending: Need to unit test on Linux.

### CODE REVIEW

**Comment by nethip** [comment]
@saurabh95 @swmitra Please review this.

**Review by swmitra — APPROVED** [review]
Good work @nethip 👍

File: src/extensions/default/HealthData/HealthDataManager.js [hunk 1]
```diff
@@ -79,7 +70,31 @@ define(function (require, exports, module) {
                         oneTimeHealthData.bracketsTheme = bracketsTheme;
                     })
                     .always(function () {
-                        return result.resolve(oneTimeHealthData);
+                        var userUuid = PreferencesManager.getViewState("UUID");
+
+                        if (!userUuid) {
+
+                            // For first launch, we are now going to
+                            // rely on the macine Hash.
```
Comment (petetnt) [inline src/extensions/default/HealthData/HealthDataManager.js]: Nit: typo `machine`

**Review by petetnt — APPROVED** [review]
LGTM with a small typo

**Pull request merged**